test/cpp/end2end/end2end_test.cc

```diff
@@ -218,8 +226,39 @@ class TestServiceImpl : public ::grpc::testing::EchoTestService::Service {
   explicit TestServiceImpl(const grpc::string& host)
```

Comment:
Ah, I think I know which one caused it. Let me try to fix it soon.

On Tue, Feb 2, 2016 at 2:02 PM, Sree Kuchibhotla <notifications@github.com>
wrote:

> In test/cpp/end2end/end2end_test.cc
> <https://github.com/grpc/grpc/pull/4958#discussion_r51643228>:
>
> > @@ -218,8 +226,39 @@ class TestServiceImpl : public ::grpc::testing::EchoTestService::Service {
> >    explicit TestServiceImpl(const grpc::string& host)
>
> hmm..this is weird..I see that you did move TestServiceImpl to a separate
> file in SHA: d6f7d99
> <https://github.com/grpc/grpc/commit/d6f7d99dc655151f3705b816de5fdac7011acc87>
> but somehow, in the latest mater, it is back in end2end_test.cc
>
> https://github.com/grpc/grpc/blob/master/test/cpp/end2end/end2end_test.cc
>
> I guess that is why I was confused :). I am trying to see where this got
> added back but so far couldn't trace it (git blame on end2end_test.cc isn't
> helping either)..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/4958/files#r51643228>.
>